### ISSUE TITLE
[Feature Request] Add a flag to disable blue/red color of buttons

### ISSUE THREAD

**Issue by valyy151** [Alex313031/thorium#307]
It would be nice if there was a flag to disable the blue/red color of buttons like the home, refresh, extensions and customize buttons.

Nothing wrong with the colors, but for some people appearance means a lot, even tiny things like this.

Other than that, this browser is a fantastic!

**Comment by Huskydude422** [comment]
I also think this is a good idea, the only thing keeping me on the browser right now is being able to switch to the new chromium theme where the colored buttons haven't been added yet

**Comment by gz83** [comment]
thanks for your advice

**Comment by Vibhav201** [comment]
Yes, I only have two reasons Thorium can't be my primary browser.
1. The ugly icon colors
2. No repository for my distro / No Flatpak availability.

**Comment by yokoffing** [comment]
Yes, please let us use the colors of the default themes.

**Comment by Nekosis** [comment]
I do agree that this should definitely be an option at the very least. Chrome themes were not designed with these colors in mind, so they clash with most of them.

**Comment by ionutbortis** [comment]
+1 just installed and configured Thorium and the first thing I googled is how to change the blue button accents :sweat_smile:

**Comment by mezhgano** [comment]
+1
Please change these icons color to default or let people disable custom colors, this looks kind of inconsistent for some people, include me

![Screenshot 2023-11-04 19-28-21  Feature_Request _Add_a_flag_to_disable_bluered_co](https://github.com/Alex313031/thorium/assets/8752899/38242f48-18b0-4130-bd12-93ba081979af)

**Comment by wadegbow** [comment]
+1 

Settings to change the color or hide these icons would be a great addition.

**Comment by LoganDark** [comment]
Thorium is unusable for me because of this. :/

It makes no sense that the button colors have to be changed just because it's a chromium fork. Please revert them back or at least provide a flag to turn this off

**Comment by LoganDark** [comment]
It looks like enabling the new design refresh fixes the button colors, but that adds a dropdown arrow on the left of the tab strip which I can't seem to turn off. That's still better than the ugly buttons though.

**Comment by Alex313031** [comment]
@LoganDark @ionutbortis @wadegbow @mezhgano @thataboy @gz83 Many users have requested this, and I FINALLY did the work for it! Next release will have the flag `chrome://flags/#disable-thorium-icons`

See commit > https://github.com/Alex313031/thorium/commit/ad4ea69e7908bb0e013750db5b1d9fe8ec14f41a

**Comment by Alex313031** [comment]
@LoganDark @ionutbortis @wadegbow @mezhgano @thataboy Added in all releases now that M121 is released today. Closing issue now.

**Comment by AntoninoScordino** [comment]
Apprecietate how this was "fixed", but I honestly think that it might _still_ a bit inconsistent considering that some icons are black and white, while others are not: especially with the "bright" red cross, which makes everything a bit awkward. But if this is "non-fixable" and rather hard to further implement, I won't judge such decision. Although, such dedication to low-priorities issues should stille be admired. Cheers :-)

**Comment by Alex313031** [comment]
@AntoninoScordino

**Comment by Alex313031** [comment]
@LoganDark Do you mean the shapes of the back/forward and menu icons?

**Comment by LoganDark** [comment]
> @LoganDark Do you mean the shapes of the back/forward and menu icons?

I'm not fully sure, you just said there are still some visual changes that aren't controlled by the new flag. As long as they don't look out of place I think it should be fine, I'm just saying that in general, I don't think Thorium should have to make those kinds of visual changes purely to differentiate itself. That's usually a mistake when Thorium's UI changes aren't held to the same standard as vanilla Chromium, since it lowers the quality/consistency of the UI.

No offense of course, just prefer to keep things vanilla~

**Comment by Alex313031** [comment]
@LoganDark Well, I can make it where the flag makes those three icons use upstream.

I'm talking about these:
![Screenshot from 2024-02-20 09-25-55](https://github.com/Alex313031/thorium/assets/45863095/88e741b7-78b2-4abe-beb8-c88a5aa06f92)
![Screenshot from 2024-02-20 09-26-17](https://github.com/Alex313031/thorium/assets/45863095/42483ad0-6ff2-47f1-8e02-928ca71130fe)

And the ones im talking about that I wont change are this:

![Screenshot from 2024-02-20 09-27-12](https://github.com/Alex313031/thorium/assets/45863095/4202f8bb-63e1-4e8f-bf71-3506c497dc44)

**Comment by LoganDark** [comment]
> @LoganDark Well, I can make it where the flag makes those three icons use upstream.
> 
> I'm talking about these: ![Screenshot from 2024-02-20 09-25-55](https://private-user-images.githubusercontent.com/45863095/306321877-88e741b7-78b2-4abe-beb8-c88a5aa06f92.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg0NDMyNTMsIm5iZiI6MTcwODQ0Mjk1MywicGF0aCI6Ii80NTg2MzA5NS8zMDYzMjE4NzctODhlNzQxYjctNzhiMi00YWJlLWJlYjgtYzg4YTVhYTA2ZjkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIwVDE1MjkxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY0Yzc3OTBhMzU5MjAxNjgxNTIzY2YxMzEwYzdkZWNiZTI1ZWNkMzIwMzA5MzFiMTMwZDM1ZDkyYTExMDJlMDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.TTm4OFvmvOBVD4KqT--izszNvDZgCvE5PtVQdaZdVRY) ![Screenshot from 2024-02-20 09-26-17](https://private-user-images.githubusercontent.com/45863095/306321982-42483ad0-6ff2-47f1-8e02-928ca71130fe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg0NDMyNTMsIm5iZiI6MTcwODQ0Mjk1MywicGF0aCI6Ii80NTg2MzA5NS8zMDYzMjE5ODItNDI0ODNhZDAtNmZmMi00N2YxLThlMDItOTI4Y2E3MTEzMGZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIwVDE1MjkxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzNTU5NmNhYjRkZDVjNzdkZWIxMGQ1MDhiM2VlMmM3Yjc1NDBjOTIzOWU1NGY0ZmQxNzQyYzRmZDQ2ZmY5OTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6b2hufWWaV9iHfEIeUh2yK-tfMEyeppDquLx6RwIhP8)
> 
> And the ones im talking about that I wont change are this:
> 
> ![Screenshot from 2024-02-20 09-27-12](https://private-user-images.githubusercontent.com/45863095/306322276-4202f8bb-63e1-4e8f-bf71-3506c497dc44.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg0NDMyNTMsIm5iZiI6MTcwODQ0Mjk1MywicGF0aCI6Ii80NTg2MzA5NS8zMDYzMjIyNzYtNDIwMmY4YmItNjNlMS00ZThmLWJmNzEtMzUwNmM0OTdkYzQ0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIwVDE1MjkxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFkM2UzMjc5MzU2OGFjNGUzNmQ5MDhjNDE5ODg1ZTZiZjA3Yjk4N2YyZWE5OTczYmU0ZThhYmUwZDExOTA0ZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.I1bp1Ahg2mRjzIEPvBq3IyoV1EJv-k_JKdox919Xpr0)
> 
> However, the top three are debatable, because most people just want the color reverted, they don't care about the shape of the button. Personally, I don't really care either way about the back/forward buttons, but the main menu IMO makes alot more sense as it is a standard "hamburger menu", rather than a "three dots menu" that almost no other application uses besides chromium.

Maybe the icon shapes should be a separate flag, but yeah.

As long as the one you won't change can still have its color reverted by the flag, I think it should be fine.

**Comment by arjpar** [comment]
I would prefer the colors to be opt-in, or it could be opt-out as well, I see that as perfectly reasonable. It could be a flag.

**Comment by Flonky** [comment]
> I would prefer the colors to be opt-in, or it could be opt-out as well, I see that as perfectly reasonable. It could be a flag.

I have to agree with this. The colours are ignoring all theming and really stand out, so it should definitely at least be possible to opt-out of them completely.

**Comment by Huskydude422** [comment]
> I also think this is a good idea, the only thing keeping me on the browser right now is being able to switch to the new chromium theme where the colored buttons haven't been added yet

almost 7 months later and changing the colors back completely STILL isn't an option, and now I can't use the chrome refresh to get around the ugly colors 🙃 why even add the colors to begin with???

**Comment by highbaser** [comment]
I use Thorium with `#disable-thorium-icons` turned on.

This displays the icon set in a half-way state.
- X icons still have red color, including widget close buttons.
- Extension puzzle icon (upper right corner) is still blue.
- Other icons have no colors.

I personally like the red X icon, even if other icons don't have any colors because the red X is much more distinguishable, e.g. while a page is loading and the refresh icon is replaced to the red X icon temporarily. However, I dislike the triangle back ◁ and forward ▷ buttons. They feel so much alien from a browser's UI. At least these shapes could be filled from the inside and look like ◀ and ▶, however arrows would be much better.

@Alex313031 The flag you've already implemented should revert all changes to icons, including their shapes.
So when turned off:
- Back, forward buttons should be arrows, not triangles.
- All icons should have their original shape and no color.
- You may still keep the red color of the X icon. :wink: 

Also, if icon colors were less vivid or saturated, they would feel less intrusive.
![recommended](https://github.com/Alex313031/thorium/assets/85064755/78cbcb47-d21c-4595-b7d9-959de949c633)

However, I think the adjustment could be a more sophisticated one. I think everyone would be happy with the following choices.
![thorium-icons](https://github.com/Alex313031/thorium/assets/85064755/ea3af865-28cc-41db-9b19-d7d69dbb0073)

**Comment by ejaysss** [comment]
how to make these arrows not blue?
![Снимок экрана 2024-06-26 в 00 50 02](https://github.com/Alex313031/thorium/assets/150336162/ede17ad4-a0c2-47f0-9044-0fc759c49ea6)

**Comment by grayairr** [comment]
Some icons are still colored even after disabling them in Flags. E.g. The extensions button, the button that cancels page loading, etc.

**Comment by mezhgano** [comment]
I'm on latest release, player and tab mute icons is violet:

![Screenshot 2024-11-09 14-20-50_-_YouTube_-_Thor](https://github.com/user-attachments/assets/e968dd22-6d23-4746-807c-7b98424d6b5a)

**Comment by gz83** [comment]
In version 150, this flag controls the color of more buttons.
